Core/PARStore.h

```diff
@@ -95,17 +96,41 @@ extern NSString *PARStoreDidSyncNotification;
 // This method returns an array of PARChange instances. It should not be called from within a transaction, or it will fail.
 - (NSArray *)fetchChangesSinceTimestamp:(nullable NSNumber *)timestamp;
 
+/// This method returns an array of PARChange instances for the device identifier passed in.
+/// It should not be called from within a transaction, or it will fail.
+/// Pass in nil for the device identifier to get results for all devices.
+- (NSArray *)fetchChangesSinceTimestamp:(nullable NSNumber *)timestampLimit forDeviceIdentifier:(nullable NSString *)deviceID;
```

Comment:
I noticed the discrepancy between `timestamp` above and `timestampLimit` here, but then also noticed that in the .m file the argument is called `timestampLimit` in both cases. My feeling says it should be just `timestamp` in both cases and also in both .m and .h file, but not very attached as long as they're the same for both methods.